web/src/components/CartDropDown/CartDropDown.js

```diff
@@ -32,12 +32,13 @@ export default CartDropDown
 
 const Wrapper = styled.div`
   position: absolute;
-  margin-top: var(--size-4);
-
-  padding: var(--size-4);
+  margin-top: var(--size-3);
 
   background-color: var(--gray-1);
   border-radius: var(--radius-2);
+  box-shadow: var(--shadow-3);
+
+  padding: var(--size-3);
```

Comment:
Might be worth making a small lib of vars in index.css e.g.
```
html {
--padding: var(--size-3)
}
```

Reply:
Love the idea!

Reply:
Cool creating an issue for it

https://github.com/redwoodjs/example-store/issues/72